neutrons/SNAPRed
# ewm4057 round to 5 decimals and remove dupes in peaks

## Description of work

When running the diffcal flow sometimes PDCalibration would throw up because of a column name collision key on peak position.
This PR addresses this issue.

## Explanation of work

In addition to removing the dupes this pr also has to round the peak position to 5 decimal points because of how PDCalibration seems to handle things.

Column with name @0.48568
in this case 0.48568302651230516 and 0.48568302651230505 would be the same.

## To test

Run the Diffraction Calibration Workflow with the following params

```
run=52102, calibrant=NAC.json and Peak Intensity Threshold = 0.001.
```

Observe that on next a name collision occurs, observe of this branch is no longer does.

## Link to EWM item


[EWM#4057](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=4057)
